inference/worker/chat_chain_utils.py

```diff
@@ -71,7 +79,7 @@ def similarity(ts1: str, ts2: str) -> float:
 # TODO: Can be improved, like... try to use another pass trough LLM
 # with custom tuned prompt for fixing the formatting.
 # e.g. "This is malformed text, please fix it: {malformed text} -> FIX magic :)"
-def extract_tool_and_input(llm_output: str, ai_prefix: str) -> tuple[str, str]:
+def extract_tool_and_input(llm_output: str, ai_prefix: str) -> Tuple[str, str]:
```

Comment:
Type hints should be kept as `tuple`, not `Tuple`

Reply:
This is also done